Initial commit for Dutch translations of PokéMesh

Tashla opened the pull request:
Arrays.xml is untouched, added most translations in FAQ and Strings with
a comment for some as these were questionable.

Tashla committed:
Initial commit for Dutch translations of PokéMesh

Arrays.xml is untouched, added most translations in FAQ and Strings with
a comment for some as these were questionable.

iGio90 commented on values-nl/strings.xml:
```diff
@@ -0,0 +1,132 @@
+    <string name="pokemesh_mesh_enabled">Notitificaties ingeschakeld</string>
+    <string name="pokemesh_mesh_finding">Geen nabije Pokémon op het moment!</string>
+    <string name="pokemesh_meshed">Een %s is in de buurt!</string>
+    <string name="pokemesh_multi_meshed" formatted="false">Er zijn %s in de buurt van %s!</string> <!-- Questionable! Not sure what the last parameter is used for! -->
```
%s are strings injected by the code. In this case %s will be replaced by the Pokémon name and count!

iGio90 commented on values-nl/strings.xml:
```diff
@@ -0,0 +1,132 @@
+    <string name="pokemesh_support_us">PokéMesh steunen</string>
+    <string name="pokemesh_support_us_summary">Na iedere 30 scans een advertentie op volledig scherm tonen</string>
+    <string name="pokemesh_about">Over</string>
+    <string name="pokemesh_follow_facebook" translatable="false">/PokeMesh</string> <!-- May I suggest making these two also non-translatable? Considered that you'd like to keep a single facebook/twitter page for everything! -->
```
correct suggestion!

iGio90 commented on values-nl/strings.xml:
```diff
@@ -0,0 +1,132 @@
+    <string name="pokemesh_got_pokemesh_credentials">PokéMesh gegevens aangemaakt!</string>
+    <string name="pokemesh_club_down">Pokémon GO servers zijn momenteel offline of niet bereikbaar. Probeer het later opnieuw!</string>
+    <string name="pokemesh_no_internet">Communicatie met het netwerk mislukt. Probeer het later opnieuw!</string>
+    <string name="pokemesh_got_it">Got it!</string> <!-- Questionable! Not sure where this string is used in the app! -->
```
just a general string that will be mostly used on Alert message (if any)

iGio90 commented:
Replied to all the questions! If you have any other, let me know!
Let me also know when i can merge this ^^ Thanks!

Tashla committed:
Corrective commit for several values after checking with iGio90

Changes several strings to be more in line with it's intended use, have
left "Got it!" in as this is a more general string that is commonly used
in The Netherlands aswell.

Tashla commented:
Feel free to merge! Edited most of the questionables and left "Got it!" in as this is a more general approach aswell in the Dutch language.

Also might add as a note: I've sent you an e-mail earlier at pokemeshapp@gmail.com, just checking in whether you've received it or not ;)
